src/core/auth/OidcSimpleClient.ts

```
@@ -95,156 +100,227 @@ export class OidcSimpleClient {
+    try {
+      if (!this.tokenRefreshFlowPromise) {
+        this.tokenRefreshFlowPromise = this._tokenRefreshFlow();
+      }
+      await this.tokenRefreshFlowPromise;
+
+      if (
+        this.token &&
+        this.validateTokenState({
+          token: this.token,
+          refreshToken: this.refreshToken,
+          expiresAt: this.tokenExpirationDate!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)